bench/run-decay-comparison.sh

```
@@ -0,0 +1,162 @@
+#   P4: Must not run concurrently with Item 1 (RPU re-run).
+#
+# AGENTS.md compliance:
+#   Rule 1: Same window (31536000), cap (20), NIP-66 (liveness), --no-phase2-cache
+#   Rule 2: After completion, trace 3 values per config back to log files
+#   Rule 3: Score clearing uses ls before rm for verification
+#   Rule 4: No markers — configs run sequentially with full clearing; progress file
+#           tracks (config, profile, session) tuples for resume.
+#   Rule 5: default config (no flags) must match hardcoded 0.95/session behavior
+set -uo pipefail
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)